src/gui/sdlmain.cpp

```diff
@@ -3008,11 +3029,13 @@ void Config_Add_SDL() {
 	Pmulti->Set_help("Priority levels for dosbox. Second entry behind the comma is for when dosbox is not focused/minimized.\n"
 	                 "pause is only valid for the second entry.");
 
-	const char* actt[] = { "lowest", "lower", "normal", "higher", "highest", "pause", 0};
```

Comment:
The "default" values in the conf file are set on line 3028, which are still:
 
  `Pmulti->SetValue("higher,normal");`

These are DOSBox's the newly-genrated conf's defaults for this settings. So in this sense, "default" (as a thought exercise) imples "higher" for active, and "normal" for inactive.. just given it's behavior on a fresh-install or if these values /aren't/ set.

So to then have a setting called 'default' that behavior differently than the defaults is confusion. 

Maybe we could call this new setting: "os_controlled" or "unconstrained", or .. (? other simpler or shorter words for this intent?)

Reply:
right - how about "auto"?

Reply:
I like that!

We always "default to auto", because it's presumably a better around setting or works more often than not. 

Let's let's how auto behaves on Linux and window, and if there are no regressions suggest we make it the default.

Reply:
The OS's scheduler usually does "know best", and this also falls in-line with other applications.. they don't futz with priority levels or using elevated rights to do so.  
(Another option is just yanking all this entirely).

Reply:
> (Another option is just yanking all this entirely).

I avoided this just in case some people have a very specific hardware/OS/DOS app configuration and they know they want e.g. the lowest priority for their setup.

Reply:
I agree; and for users who already are using high-priority in dedicated setups.